### DISCUSSION TITLE
上彭博了

### DISCUSSION THREAD

**Discussion by blowfist** [WeNeedHome/SummaryOfLoanSuspension#964] [General]
https://www.bloomberg.com/news/articles/2022-07-19/xi-faces-surprise-revolt-from-china-homebuyers-seeking-justice

**Comment by Humphrey75432** [comment]
连中国地图都故意标错，点开以后我瞬间就不想看了，外媒肯定在报道事实的前提下夹带私货。没安什么好心。

**Reply by k624035508** [reply]
对对对   责任全在美方   你永远伟大光荣正确

**Reply by mrcaiandhisfriends** [reply]
地图的问题是次要矛盾，马克思主义哲学不是教了大家主要矛盾和次要矛盾的关系吗？如果说因噎废食，代入对外国媒体的成见，这只会让我们固步自封。更何况反对外媒的前提，应该是全面的了解他们，为了反对而反对、保持狭隘的立场是不可取的。彭博社关注这件事，更大的可能性是事件本身具有高度的新闻价值、经济影响力，而绝非是对中国内地的政治宣传。

**Reply by fnnnnng** [reply]
为了骂一下你这傻逼 老子特意发了十五分钟翻墙注册账号
“傻逼”

**Reply by xeon-chow** [reply]
不认识英文可以先查一下字典。![image](https://user-images.githubusercontent.com/20356690/180588794-a45a3ae9-3748-49b2-84bd-cf851d840b3c.jpeg)

**Reply by user-hook-name** [reply]
> 连中国地图都故意标错，点开以后我瞬间就不想看了，外媒肯定在报道事实的前提下夹带私货。没安什么好心。

至少報道了,比國内封殺強多少倍你説?

**Reply by wedxz** [reply]
> > 连中国地图都故意标错，点开以后我瞬间就不想看了，外媒肯定在报道事实的前提下夹带私货。没安什么好心。
> 
> 至少報道了,比國内封殺強多少倍你説?

还是领导的脸面重要，底层百姓的死活算个鸡毛啊？

**Reply by dagoohan** [reply]
小粉是這樣的
發生負面事件有人到國內互聯網求助→被屏蔽和諧
沒辦法只好發到推特 發到BBC CNN這種"境外勢力"平臺
小粉就又講因為他勾結境外勢力 給敵對國家遞刀子 是純純的"夠罕見" 所以絕不提供幫助

如果説國內媒體平臺允許討論 持續跟蹤報導解決問題 能把事件處理的很好 這時候要是境外媒體報導還添油加醋 夾帶私貨 大夥肯定是要駡他們是"干涉中國內政"讓他們"管好自己"
但是現在境內平臺相關詞條被屏蔽封殺 一些討論組和參與的個人賬號遭打壓、被封禁 那我覺得現在哪怕是之前很敵對的境外媒體 甚至是有一些放到平常看很嚴重的錯誤和夾帶私貨 衹要他們肯發聲肯報導 能確確實實起到一點對受害業主的支持和幫助 那應該暫時放下之前的仇恨 哪怕是成為暫時的朋友

B站算是有比較多愛國年輕人（甚至有點"粉紅"）的平臺了吧？ 前幾天有一個外國記者提問河南儲戶相關的視頻，評論和彈幕裡面也都是清一色的支持"境外記者"正義提問 因為微博和諧了河南儲戶的一些相關詞條 網友們去審查員不太敢言論審查的 美國駐華大使館 下面維權 平常評論區全是駡他們美狗和耗材的網友 這個時候也沒一個再罵的
我覺得真正的愛國者肯定是愛人民的 有的愛國大V看見河南儲戶事件和爛尾樓事件一言不發居然是因為他們"愛國" 更惡劣的所謂愛國者為了"愛國"不去指責有錯誤的有關部門，而是指責儲戶 指責爛尾樓業主 把所有責任全加在他們自己頭上，一整個受害者有罪論，（這種恰愛國流量的大V講這些喪失人性的話真不怕哪天鐵拳砸在自己頭上？）我認為這稱不上是真愛國，不是恰愛國飯就是無腦維護統治者

小粉這種實在沒得洗就抹黑別人還真是有夠飯圈思維，微博可以經常性的看見：汪海林批評某流量藝人娘炮 蝦子們就説他也留個辮子 説他是對家派來的黑子；鄭智化對女團改他歌詞感到氣憤，洗地粉就抹黑他想要敲詐一筆，說他炒作蹭熱度，說他歌曲遭封殺是台獨勢力 你也這樣 因為你實在是沒得洗 也不能把責任推到受害業主頭上 看見彭博社報導 你就一股腦的説他們是境外勢力 然後央媽給小粉洗腦境外媒體都是假新聞 都居心叵測 你就講他們都是假新聞 都居心叵測
那照你這種邏輯那共青團中央翻車不少吧？假新聞也有過吧？央視新聞也有講李文亮造謠，對nba賽事複播迴旋，把維他奶評選成民族品牌 那照這樣是不是可以講共青團中央和央視新聞全都是沒底線沒水平充滿謊言的下九流媒體？
最後是我用繁體字打了這段字 那麼我一定是搞華灣灣是1450是台獨份子 我還有有一段贊成了一些美國平臺和媒體，那麼我絕對是美分是美狗是耗材 所以我哪怕是好心給受害者發生 我也是別有用心 是居心叵測 是純純的陰謀

**Reply by mrcaiandhisfriends** [reply]
> 不认识英文可以先查一下字典。![image](https://user-images.githubusercontent.com/20356690/180588794-a45a3ae9-3748-49b2-84bd-cf851d840b3c.jpeg)

彭博社的图片没有任何问题，中国大陆>>>mainland China

**Reply by John-zhan** [reply]
> 连中国地图都故意标错，点开以后我瞬间就不想看了，外媒肯定在报道事实的前提下夹带私货。没安什么好心。

哈哈不认识英文，去学习吧

**Comment by wolfiesonfire** [comment]
可以看看上一篇文章，缺乏公正性

> Buyers of 35 projects across 22 cities have decided to stop paying mortgages as of July 12 due to project delays and **a drop in real estate prices**

https://www.bloomberg.com/news/articles/2022-07-13/chinese-homebuyers-across-22-cities-refuse-to-pay-mortgages

**Comment by volans19** [comment]
媒体发什么一点都不重要，只要你了解下Bloomberg是谁
